complaintdatabase/tests.py

```diff
@@ -33,11 +38,12 @@ def test_get_context_data_exist(self, mock_flag_enabled):
         self.assertTrue('total_complaints' in response.context_data.keys())
         self.assertTrue('timely_responses' in response.context_data.keys())
 
+    @skipIf(not hasattr(settings, 'STANDALONE'), "not running standlone")
```

Comment:
If for some reason we ever had `STANDALONE=False` in cfgov-refresh settings, this would get called as you're checking for the presence of the attribute, not its value. How about `not getattr(settings, 'STANDALONE', False)`?